### PR TITLE
RSS paging

### LINKED ISSUE #1831: RSS feeds paging
Currently, when querying a RSS feed you can limit the number of result through the limit= parameter (or in global settings).
However, there is no way to get the older content over that limit via RSS feed. 
It would be nice to have some paging, thanks to a ?page=x or ?paged=x parameter as in Wordpress, to make every link reachable.

Once implemented, it would be easy to add a "My favorite readings" page on a website. A kind of Shaarli that could be filled in a much frictionless way - with no comments.

### PR DESCRIPTION
**Pull request by j0k3r**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | fix #1831
| License       | MIT

RSS now support pagination.
Following https://tools.ietf.org/html/rfc5005#page-4

I also added some missing translation related to internal config page.

### CODE REVIEW

**Comment by nicosomb** [comment]
👍 
A little chapter in the documentation please? 

shrek_chat_potte

**Comment by j0k3r** [comment]
Documentation added ✅

File: docs/fr/user/configuration.rst [hunk 1]
```diff
@@ -53,6 +53,9 @@ Vous avez maintenant trois liens, un par statut : ajoutez-les dans votre agrég
 Vous pouvez aussi définir combien d'articles vous souhaitez dans vos flux RSS
 (50 est la valeur par défaut).
 
+Une pagination est aussi disponible pour ces flux. Il suffit de rajouter ``?page=2`` pour aller à la second page, par exemple.
```
Comment (nicosomb) [inline docs/fr/user/configuration.rst]: second > seconde

**Comment by nicosomb** [comment]
You can merge when builds will be ok.

**Pull request merged**